mix.exs

```diff
@@ -37,9 +37,10 @@ defmodule Exqlite.MixProject do
   defp deps do
     [
       {:db_connection, "~> 2.1"},
+      {:ex_sqlean, "~> 0.8.4", only: [:dev, :test]},
       {:elixir_make, "~> 0.6", runtime: false},
       {:ex_doc, "~> 0.24", only: :dev, runtime: false},
-      {:temp, "~> 0.4", only: [:test]}
+      {:temp, "~> 0.4", only: [:dev, :test]}
```

Comment:
Where is this used in the `dev` environment?

Reply:
I was using in the `IEx` for testing things around. Not sure if there is such a big deal between dev / test for libraries, does it make any difference if `exqlite` is included into another Elixir package / application?

Reply:
Let's leave it, I actually remember needing to do something similarly prior.